src/main/java/org/opentripplanner/netex/mapping/NetexMapper.java

```diff
@@ -84,6 +85,8 @@
      */
     private final Map<String, StopTime> stopTimesByNetexId = new HashMap<>();
 
+    private final Multimap<String, DatedServiceJourney> datedServiceJourneysBySjId = ArrayListMultimap.create();
```

Comment:
This needs to be cleared between entities. Should it be part of the `currentNetexIndex`?

Reply:
Otherwise we will try to process the DSJ:s for all groups coming after the one it belongs to

Reply:
There are several problems here, i think I will create a suggesion patch to fix this. The NetexMapper should not contain mapping state - that should be delegated. The reason is that it is complicated enough as it is, so I will see if I can refactor it.